tests/unit/StorybookPage.test.ts

```diff
@@ -12,9 +12,5 @@ describe('getStories', () => {
       { kind: "this is the story's kind", name: "this is the story's name", parameters: {} },
     ]);
     await browser.close();
-
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore TypeScript thinks these tests are Mocha, not Jest. Until we can figure out how to
-  // deconflict the global Mocha and Jest types, we'll manually ignore the error.
```

Comment:
By removing `@types/mocha`, we no longer need to deconflict it with `@types/jest`.